1 - identify organisations in the corpus/gather_texts.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>Now, I needed to create the <code>uprocessed/</code> dir and its subdirs manually (which is ok), but it would be nicer to do that automatically I think. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Okay, I automatically created it now and will incorporate this in the next PR. I also use os.path now for cross platform compatibility.